pkg/asset/cluster/aws/aws.go

```diff
@@ -125,3 +131,12 @@ func tagIamRoles(ctx context.Context, clusterID string, installConfig *installco
 func sharedTag(clusterID string) (string, string) {
 	return fmt.Sprintf("kubernetes.io/cluster/%s", clusterID), "shared"
 }
+
+func iamRoleForMachinePool(defaultAWS *awstypes.MachinePool, mp *types.MachinePool) string {
```

Comment:
I don't see this being used anywhere. Am I missing the use or for future use? or did it sneak in?

Reply:
Left in from other iterations of this solution. I will remove. Thanks.